makefiles/Makefile.docs-relational-migrator

```diff
@@ -0,0 +1,21 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+
+ifeq ($(STAGING_USERNAME),)
+	USER=$(shell whoami)
+else
+	USER=$(STAGING_USERNAME)
+endif
+
+PROJECT=docs-relational-migrator
+MUT_PREFIX ?= $(PROJECT)
+
+REPO_DIR=$(shell pwd)
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+
+get-build-dependencies: 
+	@echo "No Dependencies to fetch
```

Comment:
small suggestion since my original example probably wasn't the best 😅 
```suggestion
get-build-dependencies: 
	$(info    No dependencies to fetch)
```

Reply:
thats on me for not checking... thanks for the catch

Reply:
i opted to use echo instead of `$( )` as found in https://github.com/mongodb/docs-worker-pool/blob/meta/makefiles/Makefile.docs-csharp